src/main/java/com/example/compound/controller/Controller.java

```diff
@@ -310,7 +312,7 @@ private void caseYHelper(InOut inOut, HashMap<Person, Double> borrowedSoFar, Has
         // Otherwise, create a stand in person.
         else {
             Person standIn = this.userManager.createUser(
-                    name, 0.0, email);
+                    name, 0.0, email, ""); // TODO: PersonManager.createPerson?
```

Comment:
Agreed, we should have that function. Perhaps we can implement a builder pattern and add PremiumUser and other subclasses?